packages/hash/frontend/src/pages/[account-slug]/types/entity-type/data-type-badge.tsx

```diff
@@ -47,9 +55,12 @@ export const DataTypeBadge = ({
             padding: 1.25,
             alignItems: "center",
             ...(isArray ? { borderBottomLeftRadius: 0 } : {}),
+            width: prefixContainerWidth,
+            transition: ({ transitions }) => transitions.create("width"),
```

Comment:
I think this transition means it's possible to get into an odd state where the chip is jumping from line to line on hover. I think some right padding that is minimised when the width is expanded could address this.

https://user-images.githubusercontent.com/42802102/205361161-97d382e4-78c6-4082-a1e4-8ee89ae2440b.mov

Reply:
This could also be addressed in follow-up.

Reply:
Will do this in a follow-up